src/main/java/com/microsoft/azure/kusto/kafka/connect/sink/TableIngestionProperties.java

```diff
@@ -0,0 +1,9 @@
+package com.microsoft.azure.kusto.kafka.connect.sink;
+
+import com.microsoft.azure.kusto.ingest.IngestionProperties;
+import com.microsoft.azure.kusto.ingest.source.CompressionType;
+
+class TableIngestionProperties {
```

Comment:
not sure why compression type is part of TableIngestionProperties

Reply:
CompressionType is handled per table (or topic)

Reply:
It's strange because I would expect TableIngestionProperties to extend IngestionProperties, not contain it.

Reply:
it will require me to change the java client as all the props are private there and in order for me to extend it ill need to make it protected

Reply:
what do you say ?

Reply:
Ok, makes sense that the java ingestion properties will be private. 
If I remember correctly, the default access modifier should allow us to access "private" fields, or am I wrong?

I don't object the Idea to group these together, just that the naming is awkward because it implies inheritance